css/index.css

```diff
@@ -1,3 +1,635 @@
+	align-items: center;
+	height: 300px;
+	margin: 10px;
+	padding: 10px;
+	margin-bottom: 40px;
+	margin-top: 40px;
+}
+.Design .digital_design {
+	border: 1px solid black;
+	background-color: lightgray;
+	width: 600px;
+	height: 300px;
+}
+
+.Design .UX_UI{
+	border: 1px solid black;
+	background-color: lightgray;
+	width: 600px;
+	height: 300px;
+}
+
+.Design h2{
+	font-size: 24px;
+	margin: 10px;
+	padding:10px;
+}
+
+.Design p{
+	font-size: 16px;
+	margin: 10px;
+	padding: 10px;
+}
+
+.Design button{
+	width: 150px;
+	font-size:16px;
+	height: 50px;
+	background: white;
+	border-style: solid;
+	border-radius: 2px;
+	border-color: black;
+	margin: 10px;
+	padding: 10px;
+
+}
+
+
+
+.Marketing {
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+	height: 300px;
+	margin: 10px;
+	padding: 10px;
+	margin-bottom: 40px;
+	margin-top: 20px;
+}
+.Marketing .digital_marketing {
+	border: 1px solid black;
+	background-color: lightgray;
+	width: 600px;
+	height: 300px;
+}
+
+.Marketing .Web-dev{
+	border: 1px solid black;
+	background-color: lightgray;
+	width: 600px;
+	height: 300px;
+}
+
+.Marketing h2{
+	font-size: 24px;
+	margin: 10px;
+	padding:10px;
+}
+
+.Marketing p{
+	font-size: 16px;
+	margin: 10px;
+	padding: 10px;
+}
+
+.Marketing button{
+	width: 150px;
+	font-size:16px;
+	height: 50px;
+	background: white;
+	border-style: solid;
+	border-radius: : 2px;
+	border-color: black;
+	margin: 10px;
+	padding: 10px;
+
+}
+
+
+.App-dev {
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+	height: 300px;
+	margin: 10px;
+	padding: 10px;
+	margin-bottom: 40px;
+	margin-top: 20px;
+}
+.App-dev .ios {
+	border: 1px solid black;
+	background-color: lightgray;
+	width: 600px;
+	height: 300px;
+}
+
+.App-dev .android{
+	border: 1px solid black;
+	background-color: lightgray;
+	width: 600px;
+	height: 300px;
+}
+
+.App-dev h2{
+	font-size: 24px;
+	margin: 10px;
+	padding:10px;
+}
+
+.App-dev p{
+	font-size: 16px;
+	margin: 10px;
+	padding: 10px;
+}
+
+.App-dev button{
+	width: 150px;
+	font-size:16px;
+	height: 50px;
+	background: white;
+	border-style: solid;
+	border-radius: : 2px;
+	border-color: black;
+	margin: 10px;
+	padding: 10px;
+
+}
+
+.Services-facts{
+	border-top: 2px solid black;
+	display: flex;
+	justify-content: space-around;
+	align-items: flex-start;
+	height: 500px;
+}
+
+.Services-facts .facts{
+	margin-top: 40px;
+	width: 600px;
+}
+
+.Services-facts .services-info-img{
+	margin-top: 40px;
+	width: 600px;
+}
+.Services-facts h2{
+	font-size: 24px;
+}
+
+.Services-facts p{
+	font-size: 16px;
+}
+
+.footer {
+	display: flex;
+	height: 100px;
+	justify-content:center;
+	align-content: center;
+}
+.footer p{
+	color: lightgray;
+	width: 400px;
+	margin: 20px;
+	padding: 20px;
+}
+@media ( max-width:800px ) {
+	.container{
+	max-width: 800px;
+	}
+nav{
+	flex-wrap: wrap;
+	flex-direction: row;
+	justify-content: center;
+	align-items: center;
+}
+nav :nth-child(7) {order:1;}
```

Comment:
The logo disappears in tablet mode. Not sure what the cause of it is, through try adding `width: 100%;` or otherwise messing with the size of the image, as the image is on the page.

My console also says something about `flex-shrink`. Maybe `flex-shrink` shrank it too much?